src/core/server/saved_objects/migrationsv2/integration_tests/cleanup.test.ts

```diff
@@ -78,7 +77,7 @@ describe.skip('migration v2', () => {
       adjustTimeout: (t: number) => jest.setTimeout(t),
       settings: {
         es: {
-          license: 'trial',
+          license: 'basic',
```

Comment:
btw, we can remove it since `basic` is the default value

Reply:
That's good to know. There's no harm it keeping it though, as a reminder to use the default distro.